SearchBayesianSkopt: Config 0 Exception. Config: {'topK': 209, 'l1_ratio': 0.011269764260753398, 'alpha': 0.011248017606952841} - Exception: Traceback (most recent call last):
  File "/Users/lauraschiatti/Code/CDAE-python/src_v2/utils/ParameterTuning/SearchAbstractClass.py", line 353, in _objective_function
    result_dict, result_string, recommender_instance, train_time, evaluation_time = self._evaluate_on_validation(current_fit_parameters_dict)
  File "/Users/lauraschiatti/Code/CDAE-python/src_v2/utils/ParameterTuning/SearchAbstractClass.py", line 258, in _evaluate_on_validation
    recommender_instance, train_time = self._fit_model(current_fit_parameters)
  File "/Users/lauraschiatti/Code/CDAE-python/src_v2/utils/ParameterTuning/SearchAbstractClass.py", line 240, in _fit_model
    **self.recommender_input_args.CONSTRUCTOR_KEYWORD_ARGS)
TypeError: 'SLIM' object is not callable

SearchBayesianSkopt: Config 1 Exception. Config: {'topK': 1000, 'l1_ratio': 1e-05, 'alpha': 1.0} - Exception: Traceback (most recent call last):
  File "/Users/lauraschiatti/Code/CDAE-python/src_v2/utils/ParameterTuning/SearchAbstractClass.py", line 353, in _objective_function
    result_dict, result_string, recommender_instance, train_time, evaluation_time = self._evaluate_on_validation(current_fit_parameters_dict)
  File "/Users/lauraschiatti/Code/CDAE-python/src_v2/utils/ParameterTuning/SearchAbstractClass.py", line 258, in _evaluate_on_validation
    recommender_instance, train_time = self._fit_model(current_fit_parameters)
  File "/Users/lauraschiatti/Code/CDAE-python/src_v2/utils/ParameterTuning/SearchAbstractClass.py", line 240, in _fit_model
    **self.recommender_input_args.CONSTRUCTOR_KEYWORD_ARGS)
TypeError: 'SLIM' object is not callable

SearchBayesianSkopt: Search complete. Best config is None: None
SearchBayesianSkopt: Config 0 Exception. Config: {'topK': 209, 'l1_ratio': 0.011269764260753398, 'alpha': 0.011248017606952841} - Exception: Traceback (most recent call last):
  File "/Users/lauraschiatti/Code/CDAE-python/src_v2/utils/ParameterTuning/SearchAbstractClass.py", line 353, in _objective_function
    result_dict, result_string, recommender_instance, train_time, evaluation_time = self._evaluate_on_validation(current_fit_parameters_dict)
  File "/Users/lauraschiatti/Code/CDAE-python/src_v2/utils/ParameterTuning/SearchAbstractClass.py", line 258, in _evaluate_on_validation
    recommender_instance, train_time = self._fit_model(current_fit_parameters)
  File "/Users/lauraschiatti/Code/CDAE-python/src_v2/utils/ParameterTuning/SearchAbstractClass.py", line 240, in _fit_model
    **self.recommender_input_args.CONSTRUCTOR_KEYWORD_ARGS)
  File "/Users/lauraschiatti/Code/CDAE-python/src_v2/recommenders/Recommenders/SLIMElasticNetRecommender.py", line 34, in __init__
    super(SLIMElasticNetRecommender, self).__init__(URM_train, verbose=verbose)
  File "/Users/lauraschiatti/Code/CDAE-python/src_v2/recommenders/Recommenders/BaseSimilarityMatrixRecommender.py", line 20, in __init__
    super(BaseSimilarityMatrixRecommender, self).__init__(URM_train, verbose = verbose)
  File "/Users/lauraschiatti/Code/CDAE-python/src_v2/recommenders/Recommenders/BaseRecommender.py", line 20, in __init__
    self.URM_train.eliminate_zeros()
AttributeError: 'SLIMElasticNetRecommender' object has no attribute 'URM_train'

SearchBayesianSkopt: Config 1 Exception. Config: {'topK': 1000, 'l1_ratio': 1e-05, 'alpha': 1.0} - Exception: Traceback (most recent call last):
  File "/Users/lauraschiatti/Code/CDAE-python/src_v2/utils/ParameterTuning/SearchAbstractClass.py", line 353, in _objective_function
    result_dict, result_string, recommender_instance, train_time, evaluation_time = self._evaluate_on_validation(current_fit_parameters_dict)
  File "/Users/lauraschiatti/Code/CDAE-python/src_v2/utils/ParameterTuning/SearchAbstractClass.py", line 258, in _evaluate_on_validation
    recommender_instance, train_time = self._fit_model(current_fit_parameters)
  File "/Users/lauraschiatti/Code/CDAE-python/src_v2/utils/ParameterTuning/SearchAbstractClass.py", line 240, in _fit_model
    **self.recommender_input_args.CONSTRUCTOR_KEYWORD_ARGS)
  File "/Users/lauraschiatti/Code/CDAE-python/src_v2/recommenders/Recommenders/SLIMElasticNetRecommender.py", line 34, in __init__
    super(SLIMElasticNetRecommender, self).__init__(URM_train, verbose=verbose)
  File "/Users/lauraschiatti/Code/CDAE-python/src_v2/recommenders/Recommenders/BaseSimilarityMatrixRecommender.py", line 20, in __init__
    super(BaseSimilarityMatrixRecommender, self).__init__(URM_train, verbose = verbose)
  File "/Users/lauraschiatti/Code/CDAE-python/src_v2/recommenders/Recommenders/BaseRecommender.py", line 20, in __init__
    self.URM_train.eliminate_zeros()
AttributeError: 'SLIMElasticNetRecommender' object has no attribute 'URM_train'

SearchBayesianSkopt: Search complete. Best config is None: None
SearchBayesianSkopt: Config 0 Exception. Config: {'topK': 405, 'l1_ratio': 0.18535887878098506, 'alpha': 0.7802943943709338} - Exception: Traceback (most recent call last):
  File "/Users/lauraschiatti/Code/CDAE-python/src_v2/utils/ParameterTuning/SearchAbstractClass.py", line 358, in _objective_function
    if isinstance(recommender_instance, Incremental_Training_Early_Stopping):
NameError: name 'Incremental_Training_Early_Stopping' is not defined

SearchBayesianSkopt: Config 1 Exception. Config: {'topK': 1000, 'l1_ratio': 1e-05, 'alpha': 0.001} - Exception: Traceback (most recent call last):
  File "/Users/lauraschiatti/Code/CDAE-python/src_v2/utils/ParameterTuning/SearchAbstractClass.py", line 358, in _objective_function
    # if isinstance(recommender_instance, Incremental_Training_Early_Stopping):
NameError: name 'Incremental_Training_Early_Stopping' is not defined

SearchBayesianSkopt: Search complete. Best config is None: None
